shell/platform/darwin/macos/framework/Source/FlutterMutatorView.mm

```diff
@@ -26,6 +26,20 @@ @interface FlutterMutatorView () {
 
 @end
 
+/// Superview container for platform views, to which sublayer transforms are applied.
+@interface FlutterPlatformViewContainer : NSView
+@end
+
+@implementation FlutterPlatformViewContainer
+
+- (BOOL)isFlipped {
```

Comment:
Maybe `isYFlipped` or `isVerticallyFlipped`? Someday there might be a system where the origin is in the lower right...

Reply:
This is Cocoa method that needs to be overriden to switch into a sane coordinate system...

Reply:
> Someday there might be a system where the origin is in the lower right...

Please don't give them any suggestions...